cmd/backup/config.go

```diff
@@ -47,6 +47,8 @@ type Config struct {
 	BackupExcludeRegexp           RegexpDecoder   `split_words:"true"`
 	BackupSkipBackendsFromPrune   []string        `split_words:"true"`
 	GpgPassphrase                 string          `split_words:"true"`
+	AgePassphrase                 string          `split_words:"true"`
+	AgePublicKey                  []string        `split_words:"true"`
```

Comment:
Since this is a list, should it be named `AgePublicKeys`?

Reply:
Makes sense to me; will change.

Reply:
Done: https://github.com/offen/docker-volume-backup/compare/660ddd11aba232fa5ad098e27d7b5ce570bf852d..18b7fb576bf27ba159682ace6953074fc03a70a1